lib/ansible/modules/cloud/vmware/vmware_guest_file_operation.py

```diff
@@ -286,6 +287,7 @@ def fetch(self):
             fileTransferInfo = file_manager.InitiateFileTransferFromGuest(vm=self.vm, auth=creds,
                                                                           guestFilePath=src)
             url = fileTransferInfo.url
+            url = url.replace("*", hostname)
```

Comment:
Does this work with if hostname is vCenter server ?

Reply:
@Akasurde it work like before, I test it with vcenter server, @wojciechkopciewicz test it with ESX directly (I can't do that)

Reply:
OK.